src/main/java/org/ohdsi/circe/check/checkers/Comparisons.java

```diff
@@ -31,6 +29,12 @@
 
 public class Comparisons {
 
+    private static final Map<String, Integer> TIME_UNIT_CONVERSION = new HashMap<>();
+    static {
+        TIME_UNIT_CONVERSION.put(IntervalUnit.HOUR.getName(), 60 * 60);
+        TIME_UNIT_CONVERSION.put(IntervalUnit.MINUTE.getName(), 60);
+    }
+
```

Comment:
I wouldn't do it this way, it makes it seem like it's dynamic (ie: can change at runtime). Can't this function of conversion be associated to the enum itself?